lib/msal-common/src/config/Configuration.ts

```diff
@@ -20,7 +20,7 @@ const DEFAULT_TOKEN_RENEWAL_OFFSET_SEC = 300;
  * - network: this is where you can configure network implementation.
  * - crypto: implementation of crypto functions
  */
-export type ModuleConfiguration = {
+export type Configuration = {
```

Comment:
I would call this ClientConfiguration so we can use Configuration namespace in the platform packages.

Reply:
`ClientConfiguration` is for `client` provided params like `ClientId` etc and this is `BaseConfiguration` initializing system level modules. I am open to feedback, we can talk offline.

Reply:
Then use something like BaseConfiguration or ModuleConfiguration, I want to keep changes from 1.0 to 2.0 as minimal as possible, so we should keep the name of the Configuration class to reflect as such in msal-browser. Happy to talk offline.

Reply:
It is intentional to not touch msal-browser in the same PR. I plan to follow up and group the PRs before merge and ensure linkage. The PRs are kept simple for review ease. I already am making the needed changes.

Reply:
Sorry comment is misplaced, meant for the previous section :-)

Reply:
This is addressed in a future PR